config/routes.rb

```diff
@@ -4,33 +4,34 @@
   root to: "pages#home"
 
   resources :characters, only: [:new, :show]
-  resources :journey, only: [:show], param: :slug
 
   post "/new_game", to: 'journey#create'
   delete "/end_game", to: 'journey#destroy'
   post "/continue_game", to: 'journey#continue'
+  get '/about', to: 'pages#about'
+  get '/login', to: 'sessions#new'
+  post '/login', to: 'sessions#create'
+  delete '/logout', to: 'sessions#destroy'
+  get '/dashboard', to: 'users#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.